lib/chef/resource/aws_rds_instance.rb

```diff
@@ -0,0 +1,33 @@
+require 'chef/provisioning/aws_driver/aws_resource'
+
+
+class Chef::Resource::AwsRdsInstance < Chef::Provisioning::AWSDriver::AWSResource
+  aws_sdk_type AWS::RDS::DBInstance
+
+  attribute :db_instance_identifier, kind_of: String, name_attribute: true
+
+  attribute :engine, kind_of: String
+  attribute :engine_version, kind_of: String
+  attribute :db_instance_class, kind_of: String
+  attribute :multi_az, default: false, kind_of: [TrueClass, FalseClass]
+  attribute :allocated_storage, kind_of: Integer, default: 5
+  attribute :iops, kind_of: Integer
+  attribute :publicly_accessible, kind_of: [TrueClass, FalseClass], default: false
+  attribute :master_username, kind_of: String
+  attribute :master_user_password, kind_of: String
+  attribute :db_name, kind_of: String
+  attribute :db_port, kind_of: Integer
+
```

Comment:
So, our tagging support now is kind of crappy.  Most AWS clients have different methods of tagging the resources that client is responsible for and I haven't refactored the way we do it very well.  But the first step is to add

```ruby
  # @param aws_tags [Hash] Should be a hash of keys & values to add.  Keys and values
  #        can be provided as symbols or strings, but will be stored in AWS as strings.
  attribute :aws_tags, kind_of: Hash
```

Unfortunately right now this is copied from `AWSResourceWithEntry`.  I need to clean this up and create a module that represents taggability.